packages/node/src/attributes/ApplicationInformationAttributeProvider.ts

```diff
@@ -50,6 +50,10 @@ export class ApplicationInformationAttributeProvider implements BacktraceAttribu
 
     private generateDefaultApplicationSearchPaths() {
         const possibleSourcePaths = [process.cwd()];
+        const potentialCommandLineStartupFile = process.argv[1];
+        if (potentialCommandLineStartupFile && fs.existsSync(potentialCommandLineStartupFile)) {
+            possibleSourcePaths.unshift(potentialCommandLineStartupFile);
```

Comment:
Can we `path.resolve` this path before returning it?

Reply:
For my learning, why do we need an absolute path?

Reply:
the relative path generated by the backtrace-node library might point to a different place if sdk-core starts using it.